cogs/helpermessage.py

```diff
@@ -0,0 +1,149 @@
+import discord
+import toml
+from discord.ext import commands
+from utils.commands import available_subcommands
+from utils.pagination import paginated_embed_menus, PaginationView
+from db.models import HelperMessage
+from typing import AsyncGenerator, Set, Tuple
+
+class Helpermessage(commands.Cog):
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+        self.data = toml.load('config.toml')
+        self.subjects = self.data['subjects']
+        self.description = self.data['description']
+        self.helper_roles = [self.subjects[channel] for channel in self.subjects.keys()]
+
+    @commands.hybrid_group()
+    async def helpermessage(self, ctx: commands.Context):
+        """
+        Commands for handling reminders.
+        """
+        await available_subcommands(ctx)
+
+    @helpermessage.command()
+    async def list(self, ctx: commands.Context):
+        """
+        List all active helpermessage embeds
+        """
+        embed_dict = dict(
+            title = 'List of all active helpermessages',
+            description = 'Here is a list of all active helpermessages.',
+        )
+        channel_ids = await HelperMessage.all().values_list('channel_id', flat=True)
+        helpermessage_ids = await HelperMessage.all().values_list('message_id', flat=True)
+        names = [f'<#{channel}>' for channel in channel_ids]
+        values = [f'https://discord.com/channels/{ctx.guild.id}/{channel}/{helpermessage}' for channel, helpermessage in zip(channel_ids, helpermessage_ids)]
+        embeds = paginated_embed_menus(names, values, embed_dict=embed_dict)
+        embed, view = await PaginationView(ctx, embeds).return_paginated_embed_view()
+        await ctx.send(embed=embed, view=view)
+
+    async def embed_getter(self, edited_roles: Set[discord.Role]) -> AsyncGenerator[Tuple[discord.Message, discord.Role, discord.Embed], None]:
+        for role in edited_roles:
+            for channel, roles in self.subjects.items():
+                if role.id in roles:
+                    discord_channel = await self.bot.fetch_channel(int(channel))
+                    helpermessage = await discord_channel.fetch_message(HelperMessage.get(channel_id=discord_channel.id))
+                    embed = helpermessage.embeds[0]
+                    yield helpermessage, role, embed
+
+    @commands.Cog.listener()
+    async def on_member_update(self, before: discord.Member, after: discord.Member):
+        """
+        Update helper message based on user helper/dehelper.
```

Comment:
Maybe call it the "helpermessage embed" here, for consistency with the other doctsrings.